tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -218,13 +218,32 @@ shader, this value can be used as desired.
 | out vec4 **CUSTOM3**                   |                                                        |
 +----------------------------------------+--------------------------------------------------------+
 
+.. note::
+
+    MODELVIEW_MATRIX combines both the the MODEL_MATRIX and VIEW_MATRIX and is better suited when floating point issues may arise
+
 Fragment built-ins
 ^^^^^^^^^^^^^^^^^^
 
 The default use of a Godot fragment processor function is to set up the material properties of your object
 and to let the built-in renderer handle the final shading. However, you are not required to use all
 these properties, and if you don't write to them, Godot will optimize away the corresponding functionality.
 
+Below are examples of common variables calculated using the built-ins:
+
+.. code-block:: glsl
+
+    vec3 model_world_space = MODEL_MATRIX[3].xyz // Object's world space position. This is the equivalent to global_transform.origin in GDScript.
+    // Using mat3(MODEL_MATRIX) is the same as the global_transform.basis in GDScript.
```

Comment:
This deserves its own example the same as the others. It looks out of place to have this one totally in a comment while the others have examples.

Reply:
Done!